src/persist-cli/src/maelstrom/txn_list_append_multi.rs

```diff
@@ -276,18 +276,35 @@ impl Transactor {
             // TODO: It would be lovely to do this without cloning the
             // ReadHandle, but that would require adding a method to fetch the
             // un-advanced data in some `(lower, upper)`.
-            let target_inclusive = self
+            //
+            // TODO(txn): Use the timely operator instead to get additional
+            // coverage of it. In the meantime, hack access to the non-pub
+            // unblock_read by fetching a snapshot and throwing it away. This
+            // guarantees that the data shard is readable through read_ts.
+            let data_write = self
+                .client
+                .open_writer(
+                    *data_id,
+                    Arc::new(VecU8Schema),
+                    Arc::new(UnitSchema),
+                    Diagnostics::from_purpose("txn data"),
+                )
+                .await
+                .expect("data schema shouldn't change");
+            let _ = self
```

Comment:
Not covered by any test case according to the coverage report. Can we add a test?

Reply:
Nah, I wouldn't expect you to have known!

Reply:
I [adjusted the coverage report](https://github.com/MaterializeInc/materialize/pull/22156) to no longer report findings in maelstrom files.
(By the way, the reason that the code appears uncovered in the first place is that we don't record coverage from maelstrom tests because "randomized testing probably shouldn't count for coverage, [it is] not stable".)

Reply:
Thanks!